examples/mobile/HomeApp/index.html

```diff
@@ -33,19 +33,23 @@
 					</option>
 				</select>
 			</div>
+			<div class="ui-appbar-container ui-content-area">
+				<div class="ui-icon">
+					<img src="images/icon.png"/>
+				</div>
+				<span class="ui-title" style="flex: 1">Control card 1</span>
+				<a href="#open-control-app-1" class="ui-btn" styl="flat" data-inline="true" data-icon="next"></a>
+			</div>
+			<div class="ui-appbar-container ui-content-area">
+				<div class="ui-icon">
+					<img src="images/icon.png"/>
+				</div>
+				<span class="ui-title" style="flex: 1">Control control 2</span>
```

Comment:
"control" word duplicated --> should be Control card 2

Reply:
Done.